src/harness/testcases/registration_testcase.py

```diff
@@ -569,30 +564,50 @@ def test_WINNF_FT_S_REG_14(self):
         'installationParam': device_d['installationParam']}
     del conditionals_d['installationParam']['antennaBeamwidth']
     conditionals = {'registrationData': [conditionals_d]}
-    self._sas_admin.PreloadRegistrationData(conditionals)
-
-    # Inject FCC ID's
-    self._sas_admin.InjectFccId({'fccId': device_a['fccId']})
-    self._sas_admin.InjectFccId({'fccId': device_c['fccId']})
-    self._sas_admin.InjectFccId({'fccId': device_b['fccId']})
-    self._sas_admin.InjectFccId({'fccId': device_d['fccId']})
 
     device_a['measCapability'] = []
     device_c['measCapability'] = []
     device_b['measCapability'] = []
     device_d['measCapability'] = []
 
+    config = {
+        'registrationRequest': [device_a, device_c, device_b, device_d],
+        'conditionalRegistrationData': conditionals,
+        'responseCodes': [0, 200, 200, 200]
+    }
+    with open(config_filename, 'w') as f:
+      f.write(json.dumps(config, indent=2, sort_keys=False))
+
+  @winnforum_testcase
+  def test_WINNF_FT_S_REG_14(self):
+    """Pending registration in Array request (responseCode 200)
+    The response should be FAILURE.
+    """
+    config_filename = os.path.join('testcases', 'config',
```

Comment:
How about using something like this 
`config_filename = os.path.join('testcases', 'config', '{0}.config'.format(self.id().split('.')[3]))` instead of manually putting the filename. Also when this is used we can also move the Line:589 to Line:592 to generate_REG_14_config() definition and directly returning the config?